starlite/plugins/base.py

```diff
@@ -54,3 +54,20 @@ def get_plugin_for_value(value: Any, plugins: List[PluginProtocol]) -> Optional[
 class PluginMapping(NamedTuple):
     plugin: PluginProtocol
     model_class: Any
+
+    def get_value_converted_to_model_class(
```

Comment:
> Name: you're converting a value to ?

Arbitrary value is converted to `PluginMapping.model_class`

> Also, typing is Any.

I can make `PluginMapping` a generic with `PluginProtocol`'s `ModelT` and add several overloads, but prefer to keep it simple since the function is used only with dynamic values.

Reply:
Simple is fine. My point though is that it's a model instance not model class, and the name should reflect it.

Reply:
Will `get_value_converted_to_model` do?

Reply:
`value_to_model_instsnce`?